backends/ebpf/psa/externs/ebpfPsaDigest.cpp

```diff
@@ -0,0 +1,177 @@
+/*
+Copyright 2022-present Orange
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "ebpfPsaDigest.h"
+
+#include "backends/ebpf/ebpfType.h"
+#include "backends/ebpf/psa/ebpfPsaDeparser.h"
+
+namespace EBPF {
+
+class EBPFDigestPSAValueVisitor : public CodeGenInspector {
+ protected:
+    cstring digestMapInstance;
```

Comment:
can't guess what this does based on the name.

Reply:
I've done small refactor and now instance of `EBPFDigestPSA` is used to push element to the queue instead of using this variable.